sim/Cargo.toml

```diff
@@ -5,6 +5,14 @@ authors = ["Anirudh Sivaraman <sk.anirudh@gmail.com>", "Jessica Berg <jb7399@nyu
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "sim"
+crate-type = ["rlib"]
```

Comment:
Do we even need a static library? Is access to the module not enough? Maybe make this just lib?

Reply:
Lib also works.  I changed it to that.